ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/onboarding/view/OnboardingView.kt

```diff
@@ -175,106 +171,121 @@ fun OnboardingMainContent(
         val onboardingStage = onboardingState.current.collectAsState().value
 
         when (onboardingStage) {
-            OnboardingStage.ShieldedByDefault -> ShieldedByDefault()
-            OnboardingStage.UnifiedAddresses -> UnifiedAddresses()
-            OnboardingStage.More -> More()
-            OnboardingStage.Wallet -> Wallet(
-                onCreateWallet = onCreateWallet,
-                onImportWallet = onImportWallet
-            )
+            OnboardingStage.ShieldedByDefault -> ShieldedByDefault(paddingValues)
+            OnboardingStage.UnifiedAddresses -> UnifiedAddresses(paddingValues)
+            OnboardingStage.More -> More(paddingValues)
+            OnboardingStage.Wallet -> Wallet(paddingValues)
         }
-
-        BottomNav(onboardingStage.getProgress(), onboardingState::goNext)
     }
 }
 
 @Composable
 private fun TopNavButtons(onboardingState: OnboardingState) {
-    Row {
-        val currentStage = onboardingState.current.collectAsState().value
+    val currentStage = onboardingState.current.collectAsState().value
+    if (currentStage == OnboardingStage.ShieldedByDefault) return
```

Comment:
Just a style preference, but I'd prefer to keep the curly braces for if statements like this.  Otherwise it is easy for human reviewers to miss the return statement.

Reply:
Separately, I'd like to confirm with @Geffenz that we really do want to eliminate the skip button on this first stage.  Because having skip on stage 2/3 doesn't seem that useful.